cli/tests/pcluster/cli/test_export_image_logs/TestExportImageLogsCommand/test_helper/pcluster-help.txt

```diff
@@ -33,8 +34,13 @@ options:
   -i IMAGE_ID, --image-id IMAGE_ID
                         Export the logs related to the image id provided here.
   --bucket BUCKET       S3 bucket to export image builder logs data to. It
-                        must be in the same region of the image
+                        must be in the same region of the image. If not
+                        specified, the ParallelCluster default bucket or the
+                        CustomS3Bucket (if specified in the config) will be
+                        used.
   --bucket-prefix BUCKET_PREFIX
                         Keypath under which exported logs data will be stored
                         in s3 bucket. Defaults to <image_id>-logs-<current
-                        time in the format of yyyyMMddHHmm>
+                        time in the format of yyyyMMddHHmm>. If not specify
```

Comment:
Same as above for the typo and the use of constant

Reply:
The whole sentence is
```"If not specify `--bucket` option"```
Should it be `specified`?

And This is a txt file for the unit test.